R/calc_gsw_time_series.R

```diff
@@ -23,9 +23,17 @@
 #' @examples
 #' \dontshow{
 #' mapme.biodiversity:::.copy_resource_dir(file.path(tempdir(), "mapme-data"))
+#' outdir <- file.path(tempdir(), "mapme-data")
+#' path_gsw_ts <- file.path(outdir, "gsw_time_series")
+#' gsw_fnames_short <- dir(path_gsw_ts, pattern = ".tif$")
+#' gsw_fnames_long <- sub("v5_", "VER5-0_yearlyClassification", gsw_fnames_short)
+#'
+#' invisible(file.copy(
+#'    file.path(path_gsw_ts, gsw_fnames_short),
+#'    file.path(path_gsw_ts, gsw_fnames_long)
+#' ))
 #' }
 #' \dontrun{
-#' library(sf)
 #' library(mapme.biodiversity)
```

Comment:
I'd guess attaching `sf` is required for `read_sf()` to work..

Reply:
Which it does not, so you could either merge or include that line to be consistent with other examples